tests/snapshots/tests.contrib.django.test_django_snapshots.test_asgi_200_30.snap

```diff
@@ -1,22 +1,22 @@
-[[{"name" "asgi.request"
+[[{"name" "django.request"
    "service" nil
    "resource" "GET /"
    "type" "web"
    "error" 0
    "span_id" 0
    "trace_id" 0
    "parent_id" nil
-   "start" 1626984280542777400
-   "duration" 4391200
-   "meta" {"runtime-id" "b4d1175f6d564ff9b8fcdd671b0e547f"
+   "start" 1627403710492279500
+   "duration" 5672300
+   "meta" {"runtime-id" "5e01f22f3da544e992727e70ef8c141f"
            "http.method" "GET"
            "http.url" "http://127.0.0.1:8000/"
            "http.version" "1.1"
            "asgi.version" "3.0"
            "http.status_code" "200"}
    "metrics" {"_dd.agent_psr" 1.0
               "_sampling_priority_v1" 1
-              "system.pid" 612
+              "system.pid" 2091
               "_dd.tracer_kr" 1.0}}
       {"name" "django.request"
```

Comment:
Do we want to skip starting this span when the asgi middleware has already created it?

Reply:
Snapshot was stale here because the ASGI middleware was still being installed in addition to the ASGI middleware from Django - should be updated now!